packages/sdk-components-react-radix/src/accordion.tsx

```diff
@@ -47,10 +50,19 @@ export const AccordionItem = forwardRef<
   return <Item ref={ref} value={value ?? index ?? ""} {...props} />;
 });
 
-export const AccordionHeader: ForwardRefExoticComponent<
-  Omit<ComponentProps<typeof Header>, "asChild"> &
-    RefAttributes<HTMLHeadingElement>
-> = Header;
+const defaultTag = "h3";
+type Tag = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+type Props = ComponentProps<typeof Header> & { tag?: Tag };
+export const AccordionHeader = forwardRef<HTMLHeadingElement, Props>(
+  ({ tag: legacyTag, ...props }, ref) => {
```

Comment:
Since this component never accepted tag property there is no need to handle legacy version. tag property you added in meta will add control which will write directly into this call `getTagFromProps(props)`. So just remove "tag" property from here and all will continue to work.